prowler/providers/aws/services/inspector2/inspector2_findings_exist/inspector2_findings_exist.py

```diff
@@ -0,0 +1,21 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.inspector2.inspector2_client import (
+    inspector2_client,
+)
+
+
+class inspector2_findings_exist(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.status = "FAIL"
+        report.status_extended = "There are no Inspector2 findings."
+        report.resource_id = "Inspector2"
+        report.region = inspector2_client.region
```

Comment:
Is this a global service?

If not please use one of the findings region instead of the one we have as backup.